modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/resources/META-INF/custom-sql/default.xml

```diff
@@ -83,6 +100,34 @@
 				DDMFormInstance.formInstanceId DESC
 		]]>
 	</sql>
+	<sql id="com.liferay.dynamic.data.mapping.service.persistence.DDMFormInstanceFinder.findByC_G_N_D_S">
+		<![CDATA[
+			SELECT
+				{DDMFormInstance.*}
+			FROM
+				(
+					SELECT
+						DISTINCT DDMFormInstance.formInstanceId
+					FROM
+						DDMFormInstance, DDMFormInstanceVersion
+					WHERE
+						(DDMFormInstance.formInstanceId = DDMFormInstanceVersion.formInstanceId) AND
+						(DDMFormInstance.companyId = ?) AND
+						(DDMFormInstance.groupId = ?) AND
+						(DDMFormInstanceVersion.status = ?) AND
+						(
+							(LOWER(DDMFormInstance.name) LIKE ? [$AND_OR_NULL_CHECK$]) [$AND_OR_CONNECTOR$]
+							(DDMFormInstance.description LIKE ? [$AND_OR_NULL_CHECK$])
+						)
+				)
+				TEMP_TABLE
```

Comment:
@linolaoj, can you please clarify why did you need to use a `TEMP_TABLE` here?

Reply:
@natocesarrego , this query is supposed to be the same of com.liferay.dynamic.data.mapping.service.persistence.DDMFormInstanceFinder.findByC_G_N_D but including the DDMFormInstanceVersion table to check the status. I just didn't want to change the original query.